bom/pom.xml

```diff
@@ -91,7 +91,7 @@
             <dependency>
                 <groupId>org.jdbi</groupId>
                 <artifactId>jdbi3-oracle12</artifactId>
-                <version>${project.version}</version>
+                <version>3.5.2</version>
```

Comment:
If we do decide to continue it as a separate project, should we just remove this management entirely rather than pinning it?

Reply:
Removed